sale_channel_hook_stock_variation/models/sale_channel.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleChannel(models.Model):
+    _inherit = "sale.channel"
+
+    hook_active_stock_variation = fields.Boolean("Active stock variation hook")
+    warehouse_id = fields.Many2one("stock.warehouse")
```

Comment:
We also need to define which stock we want to export : https://github.com/shopinvader/odoo-shopinvader/blob/14.0/shopinvader_product_stock/models/shopinvader_backend.py#L27

Reply:
done